tool/evaluate-server.js

```diff
@@ -175,6 +175,12 @@ module.exports = {
             default: 0,
             help: 'Start evaluation from this line of input data',
         });
+        parser.add_argument('--output-errors', {
+            required: false,
+            default: process.stdout,
+            type: fs.createWriteStream,
+            help: "Write erred examples to this file instead of stdout"
```

Comment:
Btw "erred" is not a word, the word is "erroneous" (or "bogus" which is more common)